.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ deployment:
     branch: master
     commands:
       - >
-        digdag push $(basename $(git rev-parse --show-toplevel))
+        digdag push $WF_PROJECT
```

Comment:
Docs uses `digdag push $(basename $(git rev-parse --show-toplevel))`
https://tddocs.atlassian.net/wiki/spaces/PD/pages/1085008/Continuous+Deployment+of+Workflows+with+CircleCI
![Screen Shot 2020-09-01 at 11 09 10](https://user-images.githubusercontent.com/8242734/91786872-97163380-ec43-11ea-81bd-5d3fcca9731f.png)

However, this repo use `$WF_PROJECT` instead.